src/packages/cli/src/__tests__/__helpers__/integrationTest.ts

```diff
@@ -0,0 +1,150 @@
+import { getGenerator, IntrospectionEngine } from '@prisma/sdk'
```

Comment:
I have factored this out because I plan to turn it into a micro lib to be shared across our cli integration tests. I realize its not quite ready for that now. This is just a first step.